hw7/task1/handlers.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+func Index(w http.ResponseWriter, r *http.Request) {
+	ans := Answer{
+		r.Host,
+		r.UserAgent(),
+		r.RequestURI,
+		r.Header,
+	}
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.WriteHeader(http.StatusOK)
+	res, err := json.Marshal(ans)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Please return an error and error code to the client instead.

Reply:
`log.Fatal` will exit from code running, so no response will be returned to client and server will exit.